.travis.yml

```diff
@@ -23,6 +23,21 @@ matrix:
   allow_failures:
     - env: lt_branch=RC_1_0 gui=true build_system=cmake
     - env: lt_branch=RC_1_0 gui=false build_system=cmake
+  include:
+    - language: python
+      python: 2.7   # legacy Python
+      addons: true  # override
+      before_install: true
+      env: build_system=Python
```

Comment:
I didn't find this on travisci doc, remove it if not required.

Reply:
Without this change, these test will run in __allow_failures__ mode which is not what we want.